packages/strapi-plugin-i18n/admin/src/components/ModalEdit/index.js

```diff
@@ -2,51 +2,109 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Modal, ModalHeader, ModalSection, ModalFooter } from 'strapi-helper-plugin';
 import { useIntl } from 'react-intl';
-import { Button, Padded } from '@buffetjs/core';
-import { Row } from 'reactstrap';
+import { Button, Label, InputText } from '@buffetjs/core';
+import Select from 'react-select';
+import { Formik } from 'formik';
 import useEditLocale from '../../hooks/useEditLocale';
 import { getTrad } from '../../utils';
 
-const ModalEdit = ({ localeToEdit, onClose }) => {
+const ModalEdit = ({ localeToEdit, onClose, locales }) => {
   const { isEditing, editLocale } = useEditLocale();
   const { formatMessage } = useIntl();
   const isOpened = Boolean(localeToEdit);
 
-  const handleEdit = () => editLocale(localeToEdit).then(onClose);
+  const handleSubmit = async ({ displayName }) => {
```

Comment:
do you still need the async keyword for this function?

Reply:
Nice catch 😊 I don't need this one 👍🏻 , thank you 🙏🏻